### PR TITLE
Hw spring 01

### PR DESCRIPTION
**Pull request by YuraMitriakhin**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -9,4 +9,82 @@
...
+                        <target>${jdk.version}</target>
+                        <encoding>${project.build.sourceEncoding}</encoding>
+                    </configuration>
+                </plugin>
+            </plugins>
+        </pluginManagement>
+    </build>
+
+
 </project>
```
Comment (IwalkInSilence) [inline pom.xml]: add empty line

**Review by IwalkInSilence — APPROVED** [review]

File: pom.xml [hunk 2]
```diff
@@ -9,4 +9,82 @@
...
+                        <source>${jdk.version}</source>
+                        <target>${jdk.version}</target>
+                        <encoding>${project.build.sourceEncoding}</encoding>
+                    </configuration>
+                </plugin>
+            </plugins>
+        </pluginManagement>
+    </build>
+
+
```
Comment (dmytro-hryhoruk) [inline pom.xml]:
```suggestion
```

File: src/main/java/com/gmail/yuramitryahin/config/AppConfig.java [hunk 3]
```diff
@@ -0,0 +1,37 @@
+package com.gmail.yuramitryahin.config;
```
Comment (dmytro-hryhoruk) [inline src/main/java/com/gmail/yuramitryahin/config/AppConfig.java:1]: com.gmail?

File: pom.xml (hunk 1 above)
Comment (dmytro-hryhoruk) [reply pom.xml]: +

**Review by dmytro-hryhoruk — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (YuraMitriakhin) [reply pom.xml]: fixed

File: src/main/java/com/gmail/yuramitryahin/config/AppConfig.java (hunk 3 above)
Comment (YuraMitriakhin) [reply src/main/java/com/gmail/yuramitryahin/config/AppConfig.java:1]: yep, I read that you can specify your mail only in the reverse order of words

Comment (YuraMitriakhin) [reply src/main/java/com/gmail/yuramitryahin/config/AppConfig.java:1]: it's default package

Comment (YuraMitriakhin) [reply src/main/java/com/gmail/yuramitryahin/config/AppConfig.java:1]: I did this in all my previous homework)

**Review by dmytro-hryhoruk — APPROVED** [review]

**Pull request merged**